components/BuildPage/Hero.jsx

```diff
@@ -21,9 +22,7 @@ const GetStarted = () => (
 
 const Explore = () => (
   <Button variant="ghostPrimary" size="xl" asChild className="grow">
-    <a href="https://registry.olas.network/ethereum/components">
-      Explore minted agents
-    </a>
+    <a href={`${REGISTRY_URL}/ethereum/components`}>Explore minted agents</a>
```

Comment:
if we say agents, should be 
```suggestion
    <a href={`${REGISTRY_URL}/ethereum/agents`}>Explore minted agents</a>
```